apps/ui-library/.storybook/main.js

```diff
@@ -12,7 +12,9 @@ const config = {
   ],
   framework: {
     name: getAbsolutePath("@storybook/vue3-vite"),
-    options: {},
+    options: {
```

Comment:
This was testing something but I plan on removing it again b/c it broke other things....